### ISSUE TITLE
@opentelemetry/instrumentation-graphql sets `graphql.operation.name` span attribute to operation type

### ISSUE THREAD

**Issue by sfishel-splunk** [open-telemetry/opentelemetry-js-contrib#847]
Please answer these questions before submitting a bug report.

### What version of OpenTelemetry are you using?

```
"@opentelemetry/api": "^1.0.2",
"@opentelemetry/core": "^1.0.1",
"@opentelemetry/instrumentation-graphql": "^0.27.1",
```

### What version of Node are you using?

v12.22.4

### Please provide the code you used to setup the OpenTelemetry SDK

I'm using `@splunk/otel`, the setup code looks like this:

```js
startTracing({
  serviceName: environment.serviceName,
  instrumentations: [
    new GraphQLInstrumentation({ mergeItems: true, depth: 1 }),
    new HttpInstrumentation({
      ignoreOutgoingUrls: [/v1\/trace/, /v2\/datapoint/, /v2\/spans/],
    }),
    new ExpressInstrumentation({ ignoreLayersType: [ExpressLayerType.MIDDLEWARE, ExpressLayerType.ROUTER] })
  ],
  spanExporterFactory: () => new ZipkinExporter({ url: endpoint }),
  tracerConfig: {
    resource: new Resource(otelServiceAndEnvironment)
  },
  propagatorFactory: () =>
    new CompositePropagator({
      propagators: [new B3Propagator(), new W3CTraceContextPropagator()]
    })
});
```

### What did you do?

I applied the above instrumentation configuration to an Apollo graphql server.

### What did you expect to see?

I expected the `graphql.operation.name` span attribute to be set to the operation name as defined [here](https://graphql.org/learn/queries/#operation-name).

### What did you see instead?

Instead the `graphql.operation.name` span attribute is set to the operation *type* (e.g. "query")

**Comment by mentos1386** [comment]
Similar to #705

**Comment by dchambers** [comment]
@dyladan / @mentos1386, would you be open to me creating a PR to fix this bug?

**Comment by dyladan** [comment]
of course

**Comment by dchambers** [comment]
@dyladan, might you be able to move the PR forward from its current state?:

![image](https://user-images.githubusercontent.com/283294/154301769-09edc35a-ebb8-4347-8d87-51b58b8b37b1.png)

**Comment by jscherer92** [comment]
@dyladan can we get this issue closed since a merge was supposed to fix this? Trying scroll through our bugs list and get it cleaned up a bit :)

Thanks!

**Comment by Flarna** [comment]
fixed by #903
